### DISCUSSION TITLE
Vue V-model

### DISCUSSION THREAD

**Discussion by jiblett1000** [chakra-ui/ark#813] [Q&A] [answered]
Hey again,

Sorry in advance for littering this repo with questions, bug reports, etc.

 It seems like v-model isn't available on the components for Vue, or at least isn't working properly yet. For example, the number input docs page lists `modelValue`, but it doesn't do anything (or maybe is the same as the `value` prop?) and `v-model` causes the input to lose the ability to input decimal points for some reason.
 
 Anyhow, just wondering if this is by design or just because Ark is still at `0.1` ... considering v-model is pretty important in terms of Vue DX.
 
 Thanks for any insight!

**Comment by Shyrro** [comment]
Hey @jiblett1000 !

Thanks for reaching out. It is not by design, v-model is available under certain components and will always be if the component needs it by design.
Do you have a list of components where it did not work well for you so that we can fix it?

Thanks !

**Comment by jiblett1000** [comment]
@Shyrro Thanks for getting back at me :)

I haven't used a lot of the components yet, just noticed it particularly with the number input where the decimal input stopped working when using v-model. But, I will start posting bug reports now that I know v-model should be available when it makes sense.

I just wasn't sure if ya'll were looking for bug reports just yet or maybe waiting for a beta or something... didn't want to clutter the repo unnecessarily.

**Comment by jiblett1000** [comment, marked as answer]
@Shyrro I created a bug report for this. https://github.com/chakra-ui/ark/issues/814
